lobby-client-http/src/main/java/org/triplea/http/client/error/report/ErrorUploadClient.java

```diff
@@ -1,15 +1,12 @@
 package org.triplea.http.client.error.report;
 
-import org.triplea.http.client.error.report.create.ErrorReport;
-import org.triplea.http.client.error.report.create.ErrorReportResponse;
-
 import com.google.common.annotations.VisibleForTesting;
 
 import feign.Headers;
 import feign.RequestLine;
 
 @SuppressWarnings("InterfaceNeverImplemented")
```

Comment:
Side Note: This seems like this should be a final class with static members only.

Reply:
The warning is a false positive as Feign auto-magic builder code does produce instances of the interface: https://github.com/triplea-game/triplea/blob/master/lobby-client-http/src/main/java/org/triplea/http/client/FeignFactory.java#L31

Reply:
Makes sense.
You should probably add this as a comment for future reference.

Reply:
If the suppression were one-off, I'd completely agree with the comment. Perhaps it would not be bad to list this on a README, I somewhat tend to think a 'suppression' glossary file with reasons and links to example suppression reasons would be a good thing. I would not want to see the same comment copy/pasted with this suppression.